### DISCUSSION TITLE
GenerateResourcesAndImage fails with error - write_contents("/root/.vcpkg/vcpkg.path.txt"): No such file or directory

### DISCUSSION THREAD

**Discussion by plynch66** [actions/virtual-environments#6600] [General questions]
Hi,

I have generated a number of images in the last few weeks successfully but in the last few days I've been getting the same error regardless of which image I try to create. The error, which seems to occur quite late in the build process is : 

"    azure-arm: vcpkg collects usage data in order to help us improve your experience.
    azure-arm: The data collected by Microsoft is anonymous.
    azure-arm: You can opt-out of telemetry by re-running the bootstrap-vcpkg script with -disableMetrics,
    azure-arm: passing --disable-metrics to vcpkg on the command line,
    azure-arm: or by setting the VCPKG_DISABLE_METRICS environment variable.
    azure-arm:
    azure-arm: Read more about vcpkg telemetry at docs/about/privacy.md
    azure-arm: write_contents("/root/.vcpkg/vcpkg.path.txt"): No such file or directory
==> azure-arm: Script exited with non-zero exit status: 1. Allowed exit codes are: [0]
==> azure-arm: Step "StepProvision" failed
==> azure-arm: [c] Clean up and exit, [a] abort without cleanup, or [r] retry step (build may fail even if retry succeeds)?"

Is anyone else seeing this error and have any idea how I can fix this ?

Paul Lynch

**Comment by mikhailkoliada** [comment]
You can create the file manually with the touch/mkdir commands. I have already reported this bug to vcpkg upstream, but still no answer. You can check the current repo main for this purpose at we have applied this as a wirkaround.
